test/perf/metrics-ios.yml

```diff
@@ -11,4 +11,4 @@ startupTimeTest:
 
 binarySizeTest:
   diffMin: 200 KiB
-  diffMax: 400 KiB
+  diffMax: 600 KiB
```

Comment:
Is there a reason for the increase?

Reply:
I haven't noticed any change that would cause it.

Here is Release 5.0.0-alpha.11 branch -> passed on size https://github.com/getsentry/sentry-react-native/pull/2389/commits/8c2c3f89c403323681e058266cb46a6857980844

Here is the same branch merged -> failed on size https://github.com/getsentry/sentry-react-native/pull/2389/commits/be4a8083ee673a21ae0e91cb7c8c79bf4efb22f5

I would say the range was set too tight.

Reply:
@vaind would be fine for you? since you came up with the numbers

Reply:
<img width="514" alt="Screenshot 2023-01-30 at 16 17 13" src="https://user-images.githubusercontent.com/31292499/215517169-19453c0a-4038-4740-b5f2-2f9bcd7bc413.png">

Here are the numbers before and after in one table for easier comparison.

Reply:
Just going back to this I've missed that `replay` is part of `browser` now.

That clearly explains the difference. 

https://github.com/getsentry/sentry-javascript/blob/d1934af091ce65ebd3ca7964d2f8e694fe2ef85a/CHANGELOG.md#L35